AutoMapper.AspNetCore.OData.EFCore/QueryableExtensions.cs

```diff
@@ -201,5 +212,175 @@ private static long QueryLongCount<TModel, TData>(this IQueryable<TData> query,
                 (
                     mapper.MapExpression<Expression<Func<TData, bool>>>(modelFilter)
                 );
+        
+        private static ICollection<Expression<Func<TSource, object>>> BuildWithLiteralLists<TSource>(this IEnumerable<List<Expansion>> includes, List<string> selects)
```

Comment:
@BlaiseD I had an issue with my previous commit, once I pass some data to `.BuildIncludes<TModel>(options.SelectExpand.GetSelects())` (to the selects) it only picks that, which is make sense, but I want it to also grab literal lists, and since this code isn't in this library (resides in LogicBuilder), I copied these parts to this repo so I can make it work.

Reply:
I'm just not sure about the approach here...